notebook/image-classification.ipynb

```diff
@@ -0,0 +1,474 @@
+{
```

Comment:
<p>Perhaps you could use the <code>gray_r</code> (gray reversed) cmap instead to make this a black/gray number on a white background?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgetown-analytics/machine-learning/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Thanks for the tip! I think that would look better. You should start a visualization library or something..